public/index.html

```diff
@@ -30,7 +30,7 @@
             const botConnection = new BotChat.DirectLine({
                 secret: params['s'],
                 token: params['t']
-                }, params["domain"]
+                }, params["domain"] //the domain should be the DL url https://directline.botframework.com/v3/directline
```

Comment:
I found no documentation on default parameters. In the script, [params is empty](https://github.com/Microsoft/BotFramework-WebChat/blob/master/public/index.html#L20) 
and if I left it unset the whole thing wouldn't work.